.env

```diff
@@ -24,8 +24,15 @@ BLOCK_HEIGHT=0
 # Query node GraphQL server port
 GRAPHQL_SERVER_PORT=8081
 
-# Query node playground subscription endpoint
-GRAPHQL_PLAYGROUND_SUBSCRIPTION_ENDPOINT=ws://localhost:8081/graphql
+# Query node playground endpoint
+GRAPHQL_PLAYGROUND_ENDPOINT=http://localhost:8081/graphql
```

Comment:
Could you clarify how this is used? If I host the query-node on a public endpoint `http://www.somewhere.com/server/graphql` does this env variable have to be updated to match ?

Reply:
We would also want to apply similar config for the hydra-indexer-gateway graphql endpoint as well?

Can we solve this through proper configuration of HTTP headers forwarding at the proxy level? So that the graphql playground can properly determine what endpoint it was visited on, rather than having to manually confiugre it through env variables? Or Does the webserver used by the playground not look at the appropriate HTTP headers?

Reply:
The variable enables overwrite of playgrounds default endpoint. It seems that this value was not optimal. I changed `http://localhost:8081/graphql` to an empty value and added a comment about this. The empty value should mean that the endpoint from the browser's URL will be used. So if a proper proxy is set up, it should work ok but can be overwritten with this variable if needed.

I'll look into how to set this value for `hydra-indexer-gateway` in case we ever needed it; I forgot about gateway before you mentioned it.